From a910778820408f293085073712b98a090dd3b28a Mon Sep 17 00:00:00 2001
From: Jekyll Wu <adaptee@gmail.com>
Date: Fri, 21 Sep 2012 18:46:28 +0800
Subject: [PATCH 4/4] Specify $HOME as working directory through the .desktop
 file

This is just a workaround for the problem of opening "~/Documents" on startup.
Although it is simple (just one line) and easy to understand, it is still a
not-so-pleasant hack which doesn't need to be added at the first place.

Note, the per-profile working directory still takes precedence if it is set

CCBUG: 302903
(cherry picked from commit 5f7b5eba568e8f4d311d9fba015e6f68478b3af6)
---
 desktop/konsole.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/desktop/konsole.desktop b/desktop/konsole.desktop
index 020519c..4f5389c 100755
--- a/desktop/konsole.desktop
+++ b/desktop/konsole.desktop
@@ -5,6 +5,7 @@ TryExec=konsole
 Icon=utilities-terminal
 X-DocPath=konsole/index.html
 Terminal=false
+Path=$HOME
 X-KDE-StartupNotify=true
 
 Name=Konsole
-- 
1.7.12

